core/src/main/resources/hudson/util/HudsonIsLoading/index_lt.properties

```diff
@@ -1,24 +1,6 @@
-# The MIT License
-#
-# Copyright (c) 2004-2010, Sun Microsystems, Inc.
-#
-# Permission is hereby granted, free of charge, to any person obtaining a copy
-# of this software and associated documentation files (the "Software"), to deal
-# in the Software without restriction, including without limitation the rights
-# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
-# copies of the Software, and to permit persons to whom the Software is
-# furnished to do so, subject to the following conditions:
-#
-# The above copyright notice and this permission notice shall be included in
-# all copies or substantial portions of the Software.
-#
-# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
-# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
-# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
-# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
-# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
-# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
-# THE SOFTWARE.
+# This file is under the MIT License by authors
 
-Please\ wait\ while\ Jenkins\ is\ getting\ ready\ to\ work=\u012Ejungti automatin\u012F atnaujinim\u0105
-Your\ browser\ will\ reload\ automatically\ when\ Jenkins\ is\ ready.=Nauja u\u017Eduotis
+# /jenkins-core/src/main/resources/hudson/util/HudsonIsLoading/index_lt.properties
```

Comment:
@daniel-beck,

Line 3 for convenience, because you can find many "index_lt.properties" files and it is not clear from first look which one you are working with when it is opened in eclipse... , some current translations are very bad so it confusing to figure out the meaning searching for path some where int title bar and etc.

Those two are not correct at all.